src/web/js-interop.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "JS/TS interop"
+short-title: JS-interop
+description: "The Dart Web platform supports JavaScript/TypeScrupt integration via package:js"
```

Comment:
Ah, I see why you were capitalizing "Web" before. I'm still leery of it, since it makes us look like we wrote this in 1995. How about:

description: "Use package:js to integrate JavaScript or TypeScript code into your Dart web app."